packages/random-source-node/package-lock.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
`package-lock.json` shouldn't exist on individual packages.
This project uses package-lock.json at root https://github.com/aws/aws-sdk-js-crypto-helpers/blob/master/package-lock.json

Reply:
Whoops. Nice callout